PhysicsTools/PatUtils/python/patPFMETCorrections_cff.py

```diff
@@ -138,6 +138,7 @@
     variation = cms.int32(0),  # If not specified, default to 0
 
     seed = cms.uint32(37428479),  # If not specified, default to 37428479
+    useDeterministicSeed = cms.bool(True),
```

Comment:
technically, default behavior should not be changed in the production release.
Although in this case this is somewhat debateable.
Let's keep it "false"

Reply:
after some more thought, the requirement for reproducibility of production results wins. So, we should keep it True.